lib/casbin-ruby/logger.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'casbin-ruby/config'
+
+module Casbin
+  module Logger
+    module_function
+
+    def info(value)
+      Config.logger.info(value)
```

Comment:
We can also use just `Config.logger` instead of `Logger` (and remove this module).
But I think it is not very important.

Reply:
No. It first simple version